tests/conftest.py

```diff
@@ -474,6 +474,7 @@ def kill(self):
             """Clean up all built VMs"""
             for vm in self.vms:
                 vm.kill()
+            shutil.rmtree(tmp_path)
```

Comment:
LGTM, but might want to adapt lines 445-446 in the docstring

Reply:
done